src/Components/Title/Title.module.css

```diff
@@ -0,0 +1,27 @@
+@font-face {
```

Comment:
all the styles in a new Styles folder out of the Components routes: src/Styles/...